core/src/packets/icmp/v6/mod.rs

```diff
@@ -215,16 +220,48 @@ impl Icmpv6Type {
 pub mod Icmpv6Types {
     use super::Icmpv6Type;
 
+    /// Message type for [`Packet Too Big`].
```

Comment:
these links don't go to the actual struct definition. same problem with the v4 variant.

Reply:
ok